### PR TITLE
Add cinder service (with CEPH backend) 

### PR DESCRIPTION
**Pull request by DTadrzak**
WIP add support for cinder serivces
User has to add ceph.conf.j2 and ceph.client.cinder.keyring.j2.

### CODE REVIEW

**Comment by DTadrzak** [comment]
@PiotrProkop check this

**Comment by DTadrzak** [comment]
@pawelsocha 
@ss7pro 
@PiotrProkop 
@lleszczu 
@Quentin-M

**Comment by lleszczu** [comment]
LGTM

File: config/cinder/defaults/main.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ cinder_logging_debug: "{{ openstack_logging_debug }}"
 
 cinder_keystone_user: "cinder"
 
-openstack_cinder_auth: "{'auth_url':'{{ openstack_auth_url }}','username':'{{ openstack_username }}','password':'{{ openstack_password }}','project_name':'{{ openstack_project_name }}'}"
+openstack_cinder_auth: "{'auth_url':'{{ openstack_auth_url }}','username':'{{ openstack_username }}','password':'{{ openstack_password }}','project_name':'{{ openstack_project_name }}','domain_name':'default'}"
```
Comment (PiotrProkop) [inline config/cinder/defaults/main.yml]: why domain_name is not templated ?

File: config/cinder/templates/cinder-db-sync.sh.j2 [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+cinder-manage db sync
+
```
Comment (PiotrProkop) [inline config/cinder/templates/cinder-db-sync.sh.j2]: why not jut place "cinder-manage db sync" in COMMAND env and get rid this file ?

File: stackanetes-services/cinder/cinder-db-sync.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+type: job
+name: cinder-db-sync
+image: cinder-api
+ceph_required: true
+command: bash /tmp/cinder-db-sync.sh
```
Comment (PiotrProkop) [inline stackanetes-services/cinder/cinder-db-sync.yml]: why not jut place "cinder-manage db sync" in COMMAND env and get rid of /tmp/cinder-db-sync.sh file?

**Comment by PiotrProkop** [comment]
LGTM

**Pull request merged**